wtransport-proto/src/frame.rs

```diff
@@ -63,7 +63,7 @@ pub type IoWriteError = bytes::IoWriteError;
 pub type FrameOwned = Frame<'static>;
 
 /// An HTTP3 [`Frame`] type.
-#[derive(Copy, Clone, Debug)]
+#[derive(Copy, Clone, Debug, PartialEq)]
```

Comment:
nit: why this change? `matches!` macros should be enough

Anyway, I am not against the change, just observing is out of the scope of this PR maybe?

Reply:
As you said, it was unnecessary.
A revised commit has been made.